common/constant/key.go

```diff
@@ -234,6 +234,7 @@ const (
 	LoggerConfigPrefix         = "dubbo.logger"
 	CustomConfigPrefix         = "dubbo.custom"
 	ProfilesConfigPrefix       = "dubbo.profiles"
+	TLSConfigPrefix            = "dubbo.tls_config"
```

Comment:
Nit: tls_config -> tls-config

Reply:
But in the dubbo-go-example tls demo, it is defined as 'tls_config', which one should i choice?
https://github.com/apache/dubbo-go-samples/blob/master/tls/grpc/go-client/conf/dubbogo.yml#L9

Reply:
Ok, just keep it.